### PR TITLE
added  new link

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report #3012, so i added correct link.

### CODE REVIEW

File: docset/winserver2022-ps/international/New-WinUserLanguageList.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ New-WinUserLanguageList [-Language] <String> [<CommonParameters>]
 ## DESCRIPTION
 The **New-WinUserLanguageList** cmdlet creates a user language list object.
 The object settings include input method, spelling setting, text prediction setting, and handwriting input mode.
-For more information, see [CultureInfo Class](https://go.microsoft.com/fwlink/?LinkID=242306) and [Configurable Language and Cultural Settings](https://go.microsoft.com/fwlink/?LinkID=242307).
+For more information, see [CultureInfo Class](https://go.microsoft.com/fwlink/?LinkID=242306) and [Configurable Language and Cultural Settings](https://docs.microsoft.com/previous-versions/office/developer/sharepoint-2010/ms425914(v=office.14)).
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/international/New-WinUserLanguageList.md:25]:
```suggestion
For more information, see [CultureInfo Class](https://go.microsoft.com/fwlink/?LinkID=242306) and [Configurable Language and Cultural Settings](/previous-versions/office/developer/sharepoint-2010/ms425914(v=office.14)).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
Hello @RAJU2529 
I am not sure about linking a SharePoint 2010 article from a Windows Server one, was that link the original destination of the now broken link?
@tseward @dariomws thoughts?

**Comment by tseward** [comment]
I would agree, I don't think this is the intended link. The SharePoint article itself contains many broken links, as well.

**Review by dariomws — COMMENTED** [review]
@get-itips @tseward agree.

**Comment by scanum** [comment]
We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Pull request closed without merging**